Sources/web3swift/EthereumAddress/EthereumAddress.swift

```diff
@@ -72,7 +72,15 @@ public struct EthereumAddress: Equatable {
         return ret
     }
 
-    public init?(_ addressString: String, type: AddressType = .normal, ignoreChecksum: Bool = false) {
+    public static func contractDeploymentAddress() -> EthereumAddress {
+        return EthereumAddress("0x", type: .contractDeployment)!
+    }
+}
+
+/// In swift structs it's better to implement initializers in extension
+/// Since it's make available syntetized initializer then for free.
+extension EthereumAddress {
+    public init?(_ addressString:String, type: AddressType = .normal, ignoreChecksum: Bool = false) {
```

Comment:
And here space is missing between the name and type of the first input argument.